client/extensions/zoninator/components/settings/zone-creator/index.jsx

```diff
@@ -1,5 +1,65 @@
-const ZoneCreator = () => {
-	return null;
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { FormSection, reduxForm } from 'redux-form';
+import { localize } from 'i18n-calypso';
+import { flowRight } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormButton from 'components/forms/form-button';
+import FormButtonsBar from 'components/forms/form-buttons-bar';
+import FormLabel from 'components/forms/form-label';
+import ReduxFormTextarea from 'components/redux-forms/redux-form-textarea';
+import ReduxFormTextInput from 'components/redux-forms/redux-form-text-input';
+import SectionHeader from 'components/section-header';
+
+const ZoneCreator = ( { translate } ) => {
+	return (
+		<div>
+			<form>
+				<FormSection name="newZone">
```

Comment:
I don't think you'll actually need to use a `FormSection` component. I only used it in WPJM because there were multiple save buttons on the same form, and clicking a particular button only saves the settings in that section. Since there's only one button here, it's better to remove it.